src/test/java/InventorySystem/TestInventory.java

```diff
@@ -65,10 +69,12 @@ void testDamage() {
     void testInventory() {
         Item sword = new Sword(1);
         Item dagger = new Dagger(2);
```

Comment:
these could be global to the class instead of declaring the exact same items over and over